JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -60,14 +60,15 @@ public async Task DispatchMessage(SQSMessage message, CancellationToken cancella
             }
             catch (MessageFormatNotSupportedException ex)
             {
-                _log.LogTrace($"Didn't handle message [{message.Body ?? string.Empty}]. No serializer setup");
+                _logger.LogTrace("Could not handle SQS message {MessageId} {MessageBody}. No serializer found",
```

Comment:
These sentences should read in a way  so when you’re reading the message with formatting applied to them they make sense without context: _”Could not handle message with Id “XXX” because a serializer for the content is not configured. Message body: “{“foo”: “bar”}””_ or similar, whose template would be something like: _”Could not handle message with Id {MessageId} because a serializer for the content is not configured. Message body: “{MessageBody}””_